vendor/wp-now/src/download.ts

```diff
@@ -191,18 +191,26 @@ export async function downloadWordPress(
 	}
 }
 
-export async function downloadSqliteIntegrationPlugin() {
+export async function downloadSqliteIntegrationPlugin(
+	version = '',
+	{overwrite}: {overwrite: boolean} = {overwrite: false}
+) {
 	const finalFolder = getSqlitePath();
 	const tempFolder = path.join(os.tmpdir(), SQLITE_FILENAME);
+	const url = !! version
+		? `https://codeload.github.com/WordPress/sqlite-database-integration/zip/refs/tags/v${ version }`
+		: SQLITE_URL;
 	const { downloaded, statusCode } = await downloadFileAndUnzip({
-		url: SQLITE_URL,
+		url,
 		destinationFolder: tempFolder,
 		checkFinalPath: finalFolder,
 		itemName: 'SQLite',
+		overwrite,
 	});
 	if (downloaded) {
-		fs.ensureDirSync(path.dirname(finalFolder));
-		fs.moveSync(tempFolder, finalFolder, {
+		const nestedFolder = SQLITE_FILENAME.replace(/main$/, version);
```

Comment:
The resulting directory includes a tag suffix rather than the `-main` branch suffix. In an effort to [minimize changes](https://github.com/Automattic/studio/blob/14ef3c0f01effe46e48dd9a0b5f72f78ab0046f1/vendor/wp-now/src/constants.ts#L4) to `wp-now` source and simplify compatibility with existing installations, I opted to retain the `-main` suffix.

Reply:
With the latest changes, this was simplified to merely remove the `-main` branch suffix that is not present when downloading a release from wp.org.